LdapInterop/UserAccessAttributeParser.php

```diff
@@ -359,8 +359,8 @@ private function getNormalizedUrl($url, $isThisPiwikUrl = false)
             $parsed['port'] = 80;
         }
 
-        if (substr(@$parsed['path'], -1) !== '/') {
-            $parsed['path'] = @$parsed['path'] . '/';
+        if (!isset($parsed['path']) || substr(@$parsed['path'], -1) !== '/') {
```

Comment:
Not important
we can remove '@' here since we are checking via isset

Reply:
It can't be removed because we're executing that block when !isset

Reply:
sorry I see what you mean, can be removed

Reply:
```suggestion
        if (!isset($parsed['path']) || substr($parsed['path'], -1) !== '/') {
```